tests/test_oml/test_datasets/test_datasets.py

```diff
@@ -143,3 +152,28 @@ def test_image_datasets() -> None:
     check_base(dataset_qgl)
     check_query_gallery(dataset_qgl, df_val)
     check_labeled(dataset_qgl, df_val)
+
+
+def test_audio_datasets() -> None:
```

Comment:
pls, use `@pytest.mark.needs_optional_dependency` if you need torchaudio for this test

Reply:
done